setup.py

```diff
@@ -124,6 +124,17 @@ def getCommandLineOptions():
      "",
      "(optional) Skip nupic.core version comparison"]
   )
+  optionsDesc.append(
+    ["optimizations-native",
+    "value",
```

Comment:
@david-ragazzi i think not quiet, I want to call `./command --myOption` now I have to do `./command --myOption=True` . But `optionsDesc=[]` would not allow me to set the "myOption" name (?)